docs/guides/admin/docs/releasenotes.md

```diff
@@ -39,6 +39,10 @@ Configuration changes
 - `KARAF_NOROOT` is now set to `true` by default, preventing Opencast to be started as root user unless the
   configuration is changed.
 - The default configuration for the Paella player has been moved to `etc/ui-config/mh_default_org/paella/config.json`
+- By default, metadata catalogs and attachments sent by capture agents are discarded since these data is usually
```

Comment:
"these data are" or "this data is" (subject/verb concord). Strictly speaking data is plural, so "these data are" is more grammatically correct but most often it's used as singular ("this data is")

Reply:
I noticed a correlation between me writing docs late at night and you complaining about grammar issues ;-D
Fixed.